package.json

```diff
@@ -68,6 +68,7 @@
     "node-fetch": "^1.7.3",
     "object-hash": "^1.2.0",
     "pathway-commons": "^1.5.4",
+    "promise-simple": "^0.1.0",
```

Comment:
Why use this instead of native `Promise`s or bluebird?

Reply:
removed, changed to native Promise.